src/explore-education-statistics-common/src/modules/find-statistics/components/DataBlockTabs.tsx

```diff
@@ -55,14 +55,24 @@ const DataBlockTabs = ({
     typeof additionalTabContent === 'function'
       ? additionalTabContent({ dataBlock })
       : additionalTabContent;
-
   return (
     <LoadingSpinner loading={isLoading}>
       <Tabs id={id} testId={testId(dataBlock)} onToggle={onToggle}>
         {firstTabs}
 
         {dataBlock.charts?.length && (
```

Comment:
Not a biggy, but probably worth just adding a `!!` to make sure this coerces correctly to a boolean. In other cases, we've made the mistake of accidentally showing 0s where this check is being performed.